docs/appendix.rst

```diff
@@ -59,18 +59,14 @@ This documentation uses a few special terms to refer to Python types:
 
    WBEM connection definitions file
    connections file
-      A file maintained by pywbemcli and managed by the
-      :ref:`Connection command group`.
-      The connections file defines a list of named connections, each of which
```

Comment:
If this text from master is used as a basis, as suggested further down, change "named connections" to "named WBEM connection definitions", as agreed to in PR #295.

Reply:
Reset to master except for the change from json to yaml